engine/swarm/secrets.md

```diff
@@ -846,3 +846,80 @@ the information from a Docker-managed secret instead of being passed directly.
 >**Note**: Docker secrets do not set environment variables directly. This was a
 conscious decision, because environment variables can unintentionally be leaked
 between containers (for instance, if you use `--link`).
+
+## Using Docker Secrets in Compose
+
+For this example we will create a simple WordPress site using Docker Secrets in
+a compose file. 
+
+We will be using the short syntax that only specifies the secret 
+name. This will grant the container access to the secret and will mount its value 
+at `/run/secrets/secret_name`.
+
+More information on short and long syntax is can be found at 
+[Compose file version 3 reference](/compose/compose-file/#secrets).
+
+> **Note** The latest mySQL and WordPress images have the Docker Secret functionality 
```

Comment:
Is this important? If someone was just learning about secrets, they would probably wonder, what changed, and we don't want to get into that. So I think we can remove this note. #ShortDocsFTW